### ISSUE TITLE
Validation set and 3D front dataset preprocessing

### ISSUE THREAD

**Issue by ch3cook-fdu** [CVMI-Lab/DODA#10]
Thanks for the opensourcing of your work!

 I am not able to find the 500 ply files as mentioned DODA/data/3dfront/val_list.txt.

It would also be great if you could share the pre-processing code of 3D-front, since I encounter errors like overlapped objects and wrong size of furnitures in many scenes when processing with codes in https://github.com/3D-FRONT-FUTURE/3D-FRONT-ToolBox/tree/master

**Comment by Dingry** [comment]
Thanks for your interest in our paper. Our work focus on domain adaptation, where the model is trained on a source training set and evaluated on a target validation set. We prioritize the performance of the model on the target domain, while the performance on the source validation set is of little significance. 
Regarding the validation set for 3D-FRONT, we apologize that the data disk experienced issues, and all the data and processing scripts were lost. We are working on regenerating the data. As for some noisy scenes with unreasonable furniture, we also remain them. Since it lies in raw data, unfortunately, we are also unable to handle this issue.

**Comment by ch3cook-fdu** [comment]
Thanks for the fast reply! 

I also find that other works on 3D Front like layout generation also suffer from the noisy scenes problem.

Have you found any scripts or lists that can exclude these scenes?

**Comment by Dingry** [comment]
Hi, sorry that I haven't found any references. To address the issue of noisy furniture in rooms, I filter out such unreasonable rooms through size-based filtering. If the furniture is larger than the room size, it is likely that the room contains noisy furniture. Actually, I have reached out to 3D-FRONT regarding this issue but have not received a response yet.

**Comment by ch3cook-fdu** [comment]
I manage to solve this problem with codes from [DiffuScene](https://github.com/tangjiapeng/DiffuScene). Thanks a lot!
